src/js/service/stateService.js

```diff
@@ -24,6 +24,17 @@ stateService.setGlobalGrid = function (gridData) {
     _currentGlobalGrid = gridData;
 };
 
+stateService.getCurrentGrid = function () {
```

Comment:
this method is never used - if we remove the persistence part after toggling (see other comment) we also won't need `stateService.getCurrentGridId()`